system/Router/Router.php

```diff
@@ -672,4 +649,16 @@ protected function setDefaultController()
 
         log_message('info', 'Used the default controller.');
     }
+
+    /**
+     * @param callable|string $handler
+     *
+     * @return void
+     */
+    protected function setMatchedRoute(string $route, $handler)
```

Comment:
`: void` return can be added since it is new method.

Reply:
Thanks